CONTRIBUTING.md

```diff
@@ -12,11 +14,33 @@ If you want to support our development:
 
 Constellation is licensed under the [AGPLv3](LICENSE). When contributing, you also need to agree to our [Contributor License Agreement](https://cla-assistant.io/edgelesssys/constellation).
 
-# Developer docs
+## Reporting issues and bugs, asking questions
+
+This project uses the GitHub issue tracker. Please check the existing issues before submitting to avoid duplicates.
+
+To report a security issue, contact <security@edgeless.systems>.
```

Comment:
```suggestion
This project uses the GitHub issue tracker. Check the existing issues before submitting to avoid duplicates.

Please report any security issue via a [private GitHub vulnerability report](https://github.com/edgelesssys/constellation/security/advisories/new) or write to <security@edgeless.systems>.
```
I changed this recommendation a while ago and adapted the project's readme, but forgot to change it in the dev docs.